ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -347,6 +347,38 @@ metadata:
   name: loki-promtail
   namespace: loki
 EOF
-
+cat >> "${SHARED_DIR}/manifest_metrics.yml" << EOF
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: promtail-monitor
+  namespace: openshift-monitoring
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: promtail
+  podMetricsEndpoints:
+  - port: http-metrics
+    interval: 15s
+    scheme: http
+  namespaceSelector:
+    matchNames:
+    - loki
+EOF
+cat >> "${SHARED_DIR}/manifest_metrics_rb.yml" << EOF
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: prom-scrape-loki
+  namespace: loki
+subjects:
+  - kind: ServiceAccount
+    name: prometheus-k8s
+    namespace: openshift-monitoring
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
```

Comment:
nit. A simple custom role would also make it and be more distinct for such a purpose, e.g.:
```
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name:promtail-prometheus
rules:
- apiGroups:
  - ""
  resources:
  - pods
  verbs:
  - get
  - list
  - watch